### PR TITLE
Removes gatfruit

### PR DESCRIPTION
**Pull request by athena413**
# General Documentation

### Intent of your Pull Request
It removes gatfruit.

### Why is this change good for the game?
Mass producing nuke ops weapons is not *fun* to play against for anyone, no matter how hard it is to get.

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Gatfruit is gone

### What should players be aware of when it comes to the changes your PR is implementing?
Gatfruit is gone

### What general grouping does this PR fall under? 
Removal

### Are there any aspects of the PR that you would like us not to mention on the Wiki?
No

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
Gatfruit amount that exists: 0

# Changelog

:cl:  
rscdel: Gats gatfruit
/:cl:

### CODE REVIEW

**Comment by Ratvar** [comment]
It is not even hard, it is either Lavaland, rare and in hands of miners and ghostroles that really do not need more antag tier loot, or it is xenobio, guaranteed with time and in hands of afk people who really do not need more antag tier loot

**Comment by ChesterTheCheesy** [comment]
imagine putting thumbs up on your own PR

**Comment by UselessTheremin** [comment]
Maybe make it unable to have super high yield / make it the worst fucking thing to grow ever? Like removal isn’t a good idea because it can be improved

**Comment by UselessTheremin** [comment]
I prefer for it to be not viable rather than gone, because it is a fun thing, even if it is currently used exclusively for tiding

**Comment by athena413** [comment]
> 
> 
> I prefer for it to be not viable rather than gone, because it is a fun thing, even if it is currently used exclusively for tiding

if it makes guns it's always going to be viable because it MAKES GUNS

**Comment by Ratvar** [comment]
You just plant it everywhere, but now it is even harder to use outside of tiding, and xenobio gets it as is - so how is it better
If you wanted to balance the result, you would need to make it print a gun that is not a syndie revolver, but people that like gatfruit like what is issue about it, I suspect, and might be unsatisfied

**Comment by Jarod1200** [comment]
missed opportunity. you could have named this PR "gats gatfruit"

**Comment by maxion12345** [comment]
against just because you up voted your own pr

**Comment by jcatjmeow** [comment]
make sure its not in xenobio code

**Comment by Marmio64** [comment]
fuck you for upvoting your own PR, but this PR is still something that has been a long time coming. Gatfruit is only used to validhunt, especially by podmen from lavaland.

**Comment by Hopekz** [comment]
I really like this feature and have only seen them enhance the experience but if we have a player problem with valid hunting I can see this being an issue.

**Comment by alexkar598** [comment]
Lavaland has no reason to have it, especially a "tutorial" ghost role for botany
